test/runnable/traits.d

```diff
@@ -258,7 +258,45 @@ void test6()
 
 /********************************************************/
 
-void test7()
+void test7a()
+{
+    assert(__traits(isArray) == false);
```

Comment:
you should get a compilation error for this. Argument count checking should lead to compilation error.

Reply:
well only when there's a mismatch of course

Reply:
All these tests were copied from the `__traits(isStaticArray, T)`-tests. Shouldn't we fix this problem in another PR?

Reply:
in theory traits argument count is checked (there was an issue about this that i even opened IIRC) so maybe that this issue was not solved correctly.